cloudinary_cli/utils/api_utils.py

```diff
@@ -72,11 +72,7 @@ def download_file(remote_file, local_path, downloaded=None, failed=None):
     failed = failed if failed is not None else {}
     makedirs(path.dirname(local_path), exist_ok=True)
 
-    sign_url = True if remote_file['type'] in ("private", "authenticated") else False
-
-    download_url = cloudinary_url(asset_source(remote_file), resource_type=remote_file['resource_type'],
```

Comment:
Special character escaping is broken in the SDK so cannot use:

sign_url = remote_type['access_mode'] == 'authenticated'